sale_order_line_input/security/ir.model.access.csv

```diff
@@ -0,0 +1,5 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_window_action_form_user,access_window_action_form_user,base.model_ir_actions_act_window,sale_order_line_input.group_sale_order_access,1,0,0,0
```

Comment:
@toita86 You cannot do that as it will give you access to the whole model.

The problem is here: https://github.com/OCA/sale-workflow/blob/14.0/sale_order_line_input/models/sale_order.py#L57

You should change the code to retrieve the action throug `for_xml_id`function. Look into core code.

Reply:
Thank you for the explanation!
I have changed my approach based on your suggestion, now should be correct.